### DISCUSSION TITLE
Zlibrary 提示“Welcome to nginx!” ，请教大佬应该如何解决呢

### DISCUSSION THREAD

**Discussion by cyBian** [tw93/Pake#486] [Q&A] [answered]
打开就是下面的样子，请问这种情况应该怎么解决呢 T T 
![image](https://github.com/tw93/Pake/assets/36179618/53fb8643-1ad9-482d-aa17-72348c8d7774)

**Comment by jeasonnow** [comment, marked as answer]
zlibrary 有喝茶风险，建议使用 https://zbook.lol 替代。
